api/src/paths/dwc/submission/{datasetId}/get.ts

```diff
@@ -0,0 +1,96 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { getDBConnection } from '../../../../database/db';
+import { HTTP400 } from '../../../../errors/http-error';
+import { defaultErrorResponses } from '../../../../openapi/schemas/http-responses';
+import { authorizeRequestHandler } from '../../../../request-handlers/security/authorization';
+import { SubmissionService } from '../../../../services/submission-service';
+import { getLogger } from '../../../../utils/logger';
+
+const defaultLog = getLogger('paths/dwc/eml/get');
+
+export const GET: Operation = [
+  authorizeRequestHandler(() => {
+    return {
+      and: [
+        {
+          discriminator: 'SystemUser'
+        }
+      ]
+    };
+  }),
+  getMetadataByDatasetId()
+];
+
+GET.apiDoc = {
+  description: 'retrieves dataset metadata within elastic search',
+  tags: ['eml'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      description: 'dataset Id.',
+      in: 'path',
+      name: 'datasetId',
+      schema: {
+        type: 'string'
+      },
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'Dataset metadata response object.',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object'
+          }
+        }
+      }
+    },
+    ...defaultErrorResponses
+  }
+};
+
+/**
+ * Retrieves dataset metadata from the submission table.
+ *
+ * @returns {RequestHandler}
+ */
+export function getMetadataByDatasetId(): RequestHandler {
+  return async (req, res) => {
+    defaultLog.debug({
```

Comment:
This looks like its from another endpoint.  

We log all request URLs automatically already, and because this is a GET with no request body, we can actually just remove the debug log altogether.

A possible techdebt item for later would be to clean these up a bit in general. Any GET endpoints could forgo any logging as the URL would get logged already.

The logging I mention happens here for those curious: https://github.com/bcgov/biohubbc-platform/blob/dev/api/src/app.ts#L29